image_transport/include/image_transport/simple_publisher_plugin.hpp

```diff
@@ -118,13 +118,21 @@ class SimplePublisherPlugin : public PublisherPlugin
     const std::string & base_topic,
     rmw_qos_profile_t custom_qos,
     rclcpp::PublisherOptions options) override
+  {
+    advertiseImpl(node, base_topic, custom_qos, options);
```

Comment:
Forward to new signature here
```suggestion
    advertiseImpl(node, base_topic,
        rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(custom_qos), custom_qos), options);
```